translations/fr.json

```
@@ -216,6 +215,7 @@
   "editor.record.form.field.constraintsShortcuts": "",
   "editor.record.form.field.contacts.noContact": "Veuillez renseigner au moins un point de contact.",
   "editor.record.form.field.contactsForResource.noContact": "Veuillez renseigner au moins un point de contact responsable de la donnée.",
+  "editor.record.form.field.draft.only.disabled": "",
```

French translation is missing. Google and I propose:
```suggestion
  "editor.record.form.field.draft.only.disabled": "Ce champ sera activé une fois les données publiées",
```